sdk/src/main/java/io/dapr/serializer/CustomizableDaprObjectSerializer.java

```diff
@@ -0,0 +1,33 @@
+package io.dapr.serializer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.client.ObjectSerializer;
+import io.dapr.utils.TypeRef;
+
+import java.io.IOException;
+import java.util.Objects;
+
+public class CustomizableDaprObjectSerializer implements DaprObjectSerializer {
+
+  private final ObjectSerializer delegate;
+
```

Comment:
I would rename this one to `CustomizableObjectSerializer` and as I already mentioned I think we can use an implementation like this:
```java
public class CustomizableObjectSerializer extends ObjectSerializer implements DaprObjectSerializer {

  private final ObjectMapper objectMapper;

  public CustomizableObjectSerializer(ObjectMapper objectMapper) {
    this.objectMapper = objectMapper;
  }

  @Override
  protected ObjectMapper getObjectMapper() {
    return objectMapper;
  }
  
  @Override
  public String getContentType() {
    return "application/json";
  }
}
```